src/LercLib/BitStuffer2.cpp

```diff
@@ -174,6 +174,8 @@ bool BitStuffer2::Decode(const Byte** ppByte, size_t& nBytesRemaining, vector<un
   unsigned int numElements = 0;

```

Comment:
shouldn't this be made a type `size_t` as well?  otherwise it's possible that the check on line 177 will never be true.

Reply:
Within the Lerc codebase this function is typically called to fill an 8x8 or 16x16 block of pixels. So maxElementCount is rarely > 256. For the type size_t or uint or ushort would all work fine. Size_t is for sure big enough and allows to put dataVec.size() as a function arg w/o a cast.